src/notification-groups/about.md

```diff
@@ -57,6 +58,7 @@ cargo run add-person $your_user_name
 
 Example PRs:
 
+* [Example of adding yourself to the Apple group.](https://github.com/rust-lang/team/pull/1434)
```

Comment:
Please add a note here that the correct file to change is now teams/apple.toml, not teams/macos.toml seen in the link.

We should swap this out with a better example once one exists of someone adding themself to apple.toml.

Reply:
I'ved added a comment here: https://github.com/rust-lang/team/pull/1434#issuecomment-2094911724